File: tests/Generator/resource/kotlin/complex/TypeSchema.kt

import com.fasterxml.jackson.annotation.*

/**
 * TypeSchema specification
 */
open class TypeSchema {
    @JsonProperty("definitions") var definitions: HashMap<String, DefinitionType>? = null
    @JsonProperty("import") var import: HashMap<String, String>? = null
    @JsonProperty("root") var root: String? = null
}

File: tests/Generator/resource/rust/complex/type_schema.rs
use serde::{Serialize, Deserialize};
use definition_type::DefinitionType;

// TypeSchema specification
#[derive(Serialize, Deserialize)]
pub struct TypeSchema {
    #[serde(rename = "definitions")]
    definitions: Option<std::collections::HashMap<String, DefinitionType>>,

    #[serde(rename = "import")]
    import: Option<std::collections::HashMap<String, String>>,

    #[serde(rename = "root")]
    root: Option<String>,

}

